juriscraper/opinions/united_states/state/tex.py

```diff
@@ -14,298 +14,104 @@
-
-                    for i in range(int(nr_of_pages.text) - 1):
-                        logger.info(f"  Getting page {i + 2}")
-                        next_page = self.find_element_by_class_name(
-                            "rgPageNext"
-                        )
-                        next_page.click()
-                        self.webdriver.implicitly_wait(5)
-
-                        text = self.webdriver.page_source
-
-                        html_tree = html.fromstring(text)
-                        html_tree.make_links_absolute(self.url)
+        # The parameters required to filter in Texas
+        self.parameters = {
+            "ctl00$ContentPlaceHolder1$SearchType": "rbSearchByDocument",
+            "ctl00$ContentPlaceHolder1$dtDocumentFrom": str(last_month),
+            "ctl00$ContentPlaceHolder1$dtDocumentFrom$dateInput": last_month_str,
+            "ctl00_ContentPlaceHolder1_dtDocumentFrom_dateInput_ClientState": date_param,
+            "ctl00$ContentPlaceHolder1$dtDocumentTo": str(today),
+            "ctl00$ContentPlaceHolder1$dtDocumentTo$dateInput": today_str,
+            "ctl00$ContentPlaceHolder1$chkListDocTypes$0": "on",
+            "ctl00$ContentPlaceHolder1$btnSearchText": "Search",
+            "__VIEWSTATE": self.html.xpath("//input[@id='__VIEWSTATE']")[
+                0
+            ].get("value"),
+            f"ctl00$ContentPlaceHolder1$chkListCourts${self.checkbox}": "on",
+        }
 
-                        remove_anchors = lambda url: url.split("#")[0]
-                        html_tree.rewrite_links(remove_anchors)
-                        html_pages.append(html_tree)
-                    self.webdriver.quit()
-            return html_pages
+    def _process_html(self) -> None:
+        if not self.test_mode_enabled():
+            # Make our post request to get our data
+            self.method = "POST"
+            self._set_parameters()
+            self.html = super()._download()
 
-    def _get_case_names(self):
-        def fetcher(url):
+        for row in self.html.xpath("//table[@class='rgMasterTable']/tbody/tr"):
             if self.test_mode_enabled():
-                return "No case names fetched during tests."
+                name = "No case names fetched during tests."
             else:
-                html_tree = self._get_html_tree_by_url(url, self.request_dict)
-                plaintiff = ""
-                defendant = ""
-                try:
-                    plaintiff = html_tree.xpath(
-                        "//text()[contains(., 'Style:')]/ancestor::div[@class='span2']/following-sibling::div/text()"
-                    )[0]
-                    defendant = html_tree.xpath(
-                        "//text()[contains(., 'v.:')]/ancestor::div[@class='span2']/following-sibling::div/text()"
-                    )[0]
-                except IndexError:
-                    logger.warning(f"No title or defendant found for {url}")
```

Comment:
Looks like we lost this warning. Intentional?

Reply:
hmmm.... perhaps I have higher expectations for my code?  no.  no reason.  I just also dont see it occurring yet.

Reply:
Warnings get us emails from Sentry, so we should keep 'em and use 'em liberally.

Reply:
√